### PR TITLE
bears:  Add requirements

### PR DESCRIPTION
**Pull request by Asnelchristian**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack 991ce0863dd177c6d3e3f6c90d3c5952d0b35d45

**Comment by Adrianzatreanu** [comment]
i think you could be more concise with the commit message on the 2nd one: "R Bears: Add requirements" or smt like this?

**Comment by Adrianzatreanu** [comment]
also needs rebase once im merging mine (i just commanded rultor)

File: bears/r/FormatRBear.py [hunk 1]
```diff
@@ -19,6 +20,8 @@ class FormatRBear:
     LANGUAGES = {"R"}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    REQUIREMENTS = {RscriptRequirement(package='formatR', flag='-e',
+                                       repos='http://cran.rstudio.com')}
```
Comment (Adrianzatreanu) [inline bears/r/FormatRBear.py]: its repo not repos i guess :D i made you change that i think

**Comment by Adrianzatreanu** [comment]
looks good, i hope i didnt miss anything while reviewing,
ack a1cb5c1

**Comment by Adrianzatreanu** [comment]
ack a1cb5c1

**Comment by Asnelchristian** [comment]
reack https://github.com/coala-analyzer/coala-bears/pull/648/commits/4d36b3919d58c03bc4dd1a28b94e9a178f56a629 https://github.com/coala-analyzer/coala-bears/pull/648/commits/781ead1315ab8dbac481d0f8a6bdc080dc0d8ef1

**Comment by Asnelchristian** [comment]
@rultor merge

**Pull request merged**